src/azure-cli/azure/cli/command_modules/container/_params.py

```diff
@@ -85,7 +80,6 @@ def load_arguments(self, _):
         c.argument('identity_role', options_list=['--role'], help="Role name or id the system assigned identity will have")
 
     with self.argument_context('container create', arg_group='Network') as c:
-        c.argument('network_profile', network_profile_type)
```

Comment:
Removing this parameter directly will cause breaking change, right? Could we hide this parameter first and use deprecate info to inform users in advance? Such as: `deprecate_info=c.deprecate(redirect='--acr-identity', hide=True)`

Reply:
Yes, it is a breaking change. We cannot deprecate the parameter because this new API version no longer supports the `network_profile` property and uses a different property instead. The two properties are incompatible and could not be used together.

The `acr-identity` parameter is unrelated to the `network_profile` and is for supporting a different feature.

Reply:
OK, could we hide `--network-profile` in the help description first, but still allow users to pass in `--network-profile`?
When users passes in `--network-profile`, we will only prompt that `--network-profile` is deprecated, and will not fail the user's execution, nor do other processing.

If so, we can change it as `c.argument('network_profile', deprecate_info=c.deprecate(hide=True))`
If not, you need to add `[BREAKING CHANGE]` to the PR title to indicate that it contains breaking change

Reply:
No, it cannot be hidden. The `network-profile` has been completely removed from this API/SDK version, so using that property in any way is no longer an option.

I have changed the title to include breaking change.